nuxeo-vision-aws/src/main/java/org/nuxeo/vision/aws/AmazonRekognitionProvider.java

```diff
@@ -110,14 +114,21 @@ protected AmazonRekognition getClient() {
                 result = client;
                 if (result == null) {
                     AmazonRekognitionClientBuilder builder = AmazonRekognitionClientBuilder.standard();
-                    builder.withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials(
-                            parameters.get(ACCESS_KEY_PARAM), parameters.get(SECRET_KEY_PARAM))));
-                    builder.withRegion(parameters.get(REGION_PARAM));
+                    if (isNotBlank(parameters.get(ACCESS_KEY_PARAM))
+                            && isNotBlank(parameters.get(SECRET_KEY_PARAM))
+                            && isNotBlank(parameters.get(REGION_PARAM))) {
```

Comment:
You should probably treat the credentials and the region independently

Reply:
:+1: